### DISCUSSION TITLE
User list does not load

### DISCUSSION THREAD

**Discussion by vura** [nilsteampassnet/TeamPass#4423] [Q&A]
Hi everyone,
I have a problem. The user list is not showing up in "Users"
I managed to configure LDAP, but the user list is not showing up.
When I click on refresh list, it keeps loading.

My Teampass version is 3.1. 2.134
I recently installed it

@nilsteampassnet, can you help? Tranks

![Captura de tela 2024-10-14 155146](https://github.com/user-attachments/assets/e8b09783-3c86-473a-8e0c-daee27c97160)

**Comment by vura** [comment]
Could anyone help?


@nilsteampassnet can you help?

**Comment by corentin-soriano** [comment]
@vura any error on the browser debug console?  Or in your server logs?

**Reply by vura** [reply]
> @vura any error on the browser debug console? Or in your server logs?

@corentin-soriano 
I had already tried other things. I hadn't done that yet.
I managed to solve the problem.
In the settings the URL was http but I was accessing it with https.
I changed from HTTP to HTTPS and was able to list the users.
Thanks. Thanks
